Tests/XCDBLDTests/BuildArgumentsSpec.swift

```diff
@@ -61,10 +61,18 @@ class BuildArgumentsSpec: QuickSpec {
 				subject.derivedDataPath = ""
 			}
 
-			itCreatesBuildArguments("includes the the toolchain", arguments: ["-toolchain", "org.swift.3020160509a"]) { subject in
+			itCreatesBuildArguments("includes the toolchain", arguments: ["-toolchain", "org.swift.3020160509a"]) { subject in
 				subject.toolchain = "org.swift.3020160509a"
 			}
 
+			itCreatesBuildArguments("includes warnings as errors OFF", arguments: ["SWIFT_TREAT_WARNINGS_AS_ERRORS=NO"]) { subject in
```

Comment:
Maybe add another test to verify that the setting is not changed if left as `nil`?

Reply:
OK, I added the nil test case, and it passes.